.github/workflows/publish-storybook.yml

```diff
@@ -0,0 +1,28 @@
+name: Build storybook & deploy
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bucket: [dev-design.va.gov, staging-design.va.gov, design.va.gov]
+    steps:
+    - uses: actions/checkout@master
+    - name: Install dependencies
+      run: yarn install
+    - name: Build Storybook
+      run: yarn build-storybook
+    - uses: jakejarvis/s3-sync-action@master
```

Comment:
Is this the recommended way to sync to S3?

Reply:
:thinking: recommended by who? It certainly seems like the action that gets the most use:

![image](https://user-images.githubusercontent.com/2008881/126241912-8cf10163-1eaa-4033-bdc3-243f449a01fe.png)

Reply:
Seems fine to me either way. The alternative is to run the command directly like so,

```yaml
    steps:
      - uses: aws-actions/configure-aws-credentials@v1
        with:
          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
          aws-region: "us-gov-west-1"
          
      - name: S3 Sync stuff
        run: |
          aws s3 sync storybook-static storybook
```

Reply:
Bill's stamp of approval is good enough for me. :make-it-so: :smile: